torch/csrc/jit/serialization/mobile_bytecode_generated.h

```diff
@@ -1,10 +1,10 @@
-// automatically generated by the FlatBuffers compiler, do not modify
+// automatically @generated by the FlatBuffers compiler, do not modify
 
 
-#ifndef FLATBUFFERS_GENERATED_MOBILEBYTECODE_TORCH_JIT_MOBILE_SERIALIZATION_H_
-#define FLATBUFFERS_GENERATED_MOBILEBYTECODE_TORCH_JIT_MOBILE_SERIALIZATION_H_
+#ifndef FLATBUFFERS_FBSOURCE_GENERATED_MOBILEBYTECODE_TORCH_JIT_MOBILE_SERIALIZATION_H_
+#define FLATBUFFERS_FBSOURCE_GENERATED_MOBILEBYTECODE_TORCH_JIT_MOBILE_SERIALIZATION_H_
 
-#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers_fbsource/flatbuffers.h"
```

Comment:
Remove changes to this file.

Reply:
Yeah this change is not intended.

Reply:
Also why this file is checked in here? Can it be created on the fly?